platform/security/platform-security-core-api/src/main/java/org/codice/ddf/security/OcspService.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.security;
+
+import java.security.cert.X509Certificate;
+
+public interface OcspService {
```

Comment:
✏️ This interface strikes me as odd because it's named `OcspService`, but has no details about an actual OCSP. I think, from a design perspective, it may make more sense to pull the `ocspServerUrl` field from the implementing class `OcspChecker` into this interface.

Reply:
I think it's better to keep the `ocspServerUrl` field in the impl close to the metatype that sets it.